vulnfeeds/vulns/vulns.go

```diff
@@ -329,3 +378,8 @@ func (v *Vulnerability) ToYAML(w io.Writer) error {
 	encoder := yaml.NewEncoder(w)
 	return encoder.Encode(v)
 }
+
+func (v *Vulnerability) ToJSON(w io.Writer) error {
+	encoder := json.NewEncoder(w)
+	return encoder.Encode(v)
```

Comment:
nit: can we just avoid the `encoder` variable completely?

Reply:
Removed the function entirely as it's not used anymore.